File: finance_01/purchase.py
# 1、先建立一个购买的类
# 2、写好购买的属性
# 3、写好购买的方法（函数）

class Purchase:
    def __init__(self, OrganCode, TradeDate, 
                    TradeTime, ApplicationCode, TaCode, ProdCode, TradeQuot, TradeCode):
        self.OrganCode = OrganCode
        self.TranDate = TranDate
        self.TranTime = TranTime
        self.ApplicationCode = ApplicationCode
        self.TaCode = TaCode
        self.ProdCode = ProdCode
        self.TradeQuot = TradeQuot
        self.TradeCode = TradeCode

    def get_series_products(self):
        '''一个客户可购买多个（多次）理财产品'''
        products = []
        # 建模

File: finance02/src/main/java/finance02/entities/Purchase.java
package entities;

import java.math.BigDecimal;
import java.time.*;

public class Purchase {
    String organCode;
	LocalDate tranDate;
	String tranTime;
	String applicationCode;
	String taCode;
	String prodCode;
	BigDecimal tradeQuot;
	String tradeCode;

	public Purchase(
		String organCode, 
		LocalDate tranDate, 
		String tranTime, 
		String applicationCode,
		String taCode,
		String prodCode,
		BigDecimal tradeQuot,
		String tradeCode
		) {
			this.organCode = organCode;
			this.tranDate = tranDate;
			this.tranTime = tranTime;
			this.applicationCode = applicationCode;
			this.taCode = taCode;
			this.prodCode = prodCode;
			this.tradeQuot = tradeQuot;
			this.tradeCode = tradeCode;
		}
	
	public String getOrganCode() {
		return organCode;
	}
	public void setOrganCode(String organCode) {
		this.organCode = organCode;
	}

	public LocalDate getTranDate() {
		return tranDate;
	}
	public void setTranDate(LocalDate tranDate) {
		this.tranDate = tranDate;
	}

	public String getTranTime() {
		return tranTime;
	}
	public void setTranTime(String tranTime) {
		this.tranTime = tranTime;
	}

	public String getApplicationCode() {
		return applicationCode;
	}
	public void setApplicationCode(String applicationCode) {
		this.applicationCode = applicationCode;
	}

	public String getTaCode() {
		return taCode;
	}
	public void setTaCode(String taCode) {
		this.taCode = taCode;
	}

	public String getProdCode() {
		return prodCode;
	}
	public void setProdCode(String prodCode) {
		this.prodCode = prodCode;
	}

	public BigDecimal getTradeQuot() {
		return tradeQuot;
	}
	public void setTradeQuot(BigDecimal tradeQuot) {
		this.tradeQuot = tradeQuot;
	}

	public String getTradeCode() {
		return tradeCode;
	}
	public void setTradeCode(String tradeCode) {
		this.tradeCode = tradeCode;
	}
}